cmake/Modules/ArrowExternalProject.cmake

```diff
@@ -25,7 +25,7 @@ set(arrow_URL https://github.com/ray-project/arrow.git)
 # WARNING: If the arrow version is updated, you need to also update the
 # SETUPTOOLS_SCM_PRETEND_VERSION version string in the ThirdpartyToolchain.cmake
 # file
-set(arrow_TAG c347cd571e51723fc8512922f1b3a8e45e45b169)
+set(arrow_TAG f8c990e4165012123e03d822c95526eaea5cbfab)
```

Comment:
Ok, I changed the link to the patch (originally I didn't because it's the same changes, but could be confusing I agree).

As for an upstream fix, I don't think it will happen any time soon. Also pytorch/tensorflow won't be fixed any time soon, see the discussion. It's really annoying, but there is not much we can do about it I think. The arguments from all sides make a lot of sense to me, probably what actually needs to happen is people need either to switch away from pip to conda, or nvidia needs to make sure that cuda is compatible with manylinux1 (or manylinux2010 in the future) to really fix this for everybody (I assume you followed the discussion in that issue and the mailing list).

Reply:
Understood.